.github/workflows/builds.yml

```diff
@@ -62,29 +69,45 @@ jobs:
         BUILD_TYPE: ["Release", "Debug"]
         PLATFORM:
           - NAME: "HOST"
-            CONTAINER: "ghcr.io/acts-project/ubuntu2004:v43"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404:68"
             OPTIONS: --preset eigen -DALGEBRA_PLUGINS_USE_SYSTEM_EIGEN3=TRUE
+            run_tests: true
           - NAME: "HOST"
-            CONTAINER: "ghcr.io/acts-project/ubuntu2004:v43"
+            CONTAINER: "ghcr.io/acts-project/ubuntu2404:68"
             OPTIONS: --preset eigen
+            run_tests: true
```

Comment:
Let's follow the capitalization used in all other variables. I.e. let's call this `RUN_TESTS`.

Reply:
Done